src/test/scala/com/ibm/m4d/mover/ConfigSuite.scala

```diff
@@ -82,4 +82,36 @@ class ConfigSuite extends AnyFlatSpec with Matchers {
     substitutedConfig.hasPath("secretProviderURL") shouldBe false
     substitutedConfig.hasPath("secretProviderRole") shouldBe false
   }
+
+  it should "parse DataFlowType correctly" in {
+    DataFlowType.parse("batch") shouldBe DataFlowType.Batch
+    DataFlowType.parse("Batch") shouldBe DataFlowType.Batch
+    DataFlowType.parse("stream") shouldBe DataFlowType.Stream
+    DataFlowType.parse("Stream") shouldBe DataFlowType.Stream
+    intercept[IllegalArgumentException](DataFlowType.parse("random"))
+  }
```

Comment:
Wouldn't  be better to normalize the input, i.e. put everything lower case ?

Reply:
`DataFlowType.parse` is putting it to lower case before doing the comparison. This tests if it's also implemented that way.